### DISCUSSION TITLE
If tiangolo/uvicorn-gunicorn-fastapi is deprecated why you are using it in template repo?

### DISCUSSION THREAD

**Discussion by MarcinSkrobczynski** [fastapi/full-stack-fastapi-template#1338] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched in the documentation/README.
- [X] I already searched in Google "How to do X" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```Text
FROM tiangolo/uvicorn-gunicorn-fastapi:python3.10
```

### Description

I observed in the docker hub that images like `tiangolo/uvicorn-gunicorn-fastapi` are deprecated.
Why are they still used in this example?

### Operating System

Linux

### Operating System Details

Dont care

### Python Version

dont care

**Comment by javadzarezadeh** [comment, marked as answer]
Hey @MarcinSkrobczynski! This issue is now addressed in the PR #1351. It now uses plain python instead of uvicorn-gunicorn-fastapi.

**Reply by MarcinSkrobczynski** [reply]
Yes, I saw that. Thanks @tiangolo for changes ;)
